Library/Homebrew/tab.rb

```diff
@@ -161,6 +166,7 @@ def self.empty
       "time" => nil,
       "source_modified_time" => 0,
       "HEAD" => nil,
+      "alias_path" => nil,
```

Comment:
I'd maybe put this in `source` below and perhaps consider overriding `path` there unless it would break existing functionality relying on this in the tab.